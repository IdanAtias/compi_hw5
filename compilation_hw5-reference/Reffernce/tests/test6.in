NIS jarlettSchohansson = 200 AGORA;
	AGORA enBaffleck = 2 NIS;
	if ((jarlettSchohansson == enBaffleck)
			and (jarlettSchohansson + enBaffleck
					== enBaffleck + jarlettSchohansson))
		print "#13) But being the one on the stand,";
	else
		print "Error #11";
	jarlettSchohansson = enBaffleck*9;
	jarlettSchohansson = jarlettSchohansson/3;
	if (jarlettSchohansson == 6 NIS)
		print "#14) I know the way go go, no one's guiding me.";
	else
		print "Error #12";
	int deonardoLicaprio = jarlettSchohansson / enBaffleck;
	if (deonardoLicaprio == 3 and jarlettSchohansson / enBaffleck == 3)
		print "#15) When time soaked with blood it turns its back,";
	else
		print "Error #13";