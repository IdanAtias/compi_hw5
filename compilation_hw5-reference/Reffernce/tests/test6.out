#13) But being the one on the stand,
#14) I know the way go go, no one's guiding me.
#15) When time soaked with blood it turns its back,
